### ISSUE TITLE
If shortcut key is not set the menu item doesn't work either

### ISSUE THREAD

**Issue by rohrlich** [goki/gide#229]
I noticed this for lookup and then tried completion and same problem exists so this is important to fix.

**Comment by rcoreilly** [comment]
Yeah I think it actually triggers by using the shortcut for these cases.  why wouldn't the shortcuts be set though?  could issue an error about that?

**Comment by rohrlich** [comment]
I'm not 100% sure. I moved over to working on my desktop machine a couple of weeks ago after using my laptop for a long time and I had old prefs. But I could see where someone might change the shortcuts and leave one unset. You do get an error in that the menu item shows that the shortcut isn't set. But in the end I just think it is a bad design to chain the action. If you added voice commands would they trigger the keyboard shortcut too? Maybe not something you want to change but ...

**Comment by rcoreilly** [comment]
yeah it just takes a bit more work to get the proper active window etc -- not a big deal but can fix later..
